[2023-12-19T23:02:22.613+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hosp_insur_claim_fraud-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_hospital_County scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-19T23:02:22.632+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hosp_insur_claim_fraud-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_hospital_County scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2023-12-19T23:02:22.632+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-12-19T23:02:22.651+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_fk_col_in_fact_table_hospital_County> on 2023-12-01 00:00:00+00:00
[2023-12-19T23:02:22.665+0000] {standard_task_runner.py:57} INFO - Started process 3723 to run task
[2023-12-19T23:02:22.665+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'hosp_insur_claim_fraud-v01.insert_fk_into_fact_table', 'insert_fk_col_in_fact_table_hospital_County', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/Hospital_Insurance_Claim_Fraud.py', '--cfg-path', '/tmp/tmp0kxkryfq']
[2023-12-19T23:02:22.674+0000] {standard_task_runner.py:85} INFO - Job 39: Subtask insert_fk_col_in_fact_table_hospital_County
[2023-12-19T23:02:22.774+0000] {task_command.py:416} INFO - Running <TaskInstance: hosp_insur_claim_fraud-v01.insert_fk_into_fact_table.insert_fk_col_in_fact_table_hospital_County scheduled__2023-12-01T00:00:00+00:00 [running]> on host 183042dfa495
[2023-12-19T23:02:22.974+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dunnbc22' AIRFLOW_CTX_DAG_ID='hosp_insur_claim_fraud-v01.insert_fk_into_fact_table' AIRFLOW_CTX_TASK_ID='insert_fk_col_in_fact_table_hospital_County' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2023-12-19T23:02:22.977+0000] {sql.py:274} INFO - Executing: -- params: table_name, dimension_col

/*
Steps 1-3 were completed in the create_dim_table.sql 
file & associated PostgresOperator
*/


-- Step 4: Add foreign key column to fact table
ALTER TABLE hosp_insur_claim_fraud_pipeline
ADD COLUMN hospital_County_id INTEGER;

-- Step 4: Update foreign key column to fact table with 
-- appropriate id (that matches with dim table)
UPDATE hosp_insur_claim_fraud_pipeline
SET hospital_County_id = dimension_table_hospital_County.id
FROM dimension_table_hospital_County
WHERE COALESCE(hosp_insur_claim_fraud_pipeline.hospital_County::text, '') = COALESCE(dimension_table_hospital_County.hospital_County::text, '');
[2023-12-19T23:02:22.992+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T23:02:23.011+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-12-19T23:02:23.021+0000] {sql.py:418} INFO - Running statement: -- params: table_name, dimension_col

/*
Steps 1-3 were completed in the create_dim_table.sql 
file & associated PostgresOperator
*/


-- Step 4: Add foreign key column to fact table
ALTER TABLE hosp_insur_claim_fraud_pipeline
ADD COLUMN hospital_County_id INTEGER;

-- Step 4: Update foreign key column to fact table with 
-- appropriate id (that matches with dim table)
UPDATE hosp_insur_claim_fraud_pipeline
SET hospital_County_id = dimension_table_hospital_County.id
FROM dimension_table_hospital_County
WHERE COALESCE(hosp_insur_claim_fraud_pipeline.hospital_County::text, '') = COALESCE(dimension_table_hospital_County.hospital_County::text, '');, parameters: None
[2023-12-19T23:03:10.613+0000] {sql.py:427} INFO - Rows affected: 770347
[2023-12-19T23:03:10.647+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hosp_insur_claim_fraud-v01.insert_fk_into_fact_table, task_id=insert_fk_col_in_fact_table_hospital_County, execution_date=20231201T000000, start_date=20231219T230222, end_date=20231219T230310
[2023-12-19T23:03:10.691+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-19T23:03:10.714+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
